alyssa_mckee/function_assignments/dic_to_tup.py

```diff
@@ -0,0 +1,13 @@
+def dic_to_tup(dict):
```

Comment:
Is this method actually converting a dictionary to a list or to a tuple. It seems that it's converting it to a list.

Reply:
You're right whoops... ^_^; I was a little confused over what tuples were at that point of doing the assignments because I'd never encountered them before. I know how they work now though. I'll go back and fix it